src/commands/archive-card.js

```diff
@@ -0,0 +1,178 @@
+"use strict";
+
+var List = require('term-list');
+  var prompt = require('prompt');
+
+var __ = function(program, output, logger, config, trello, translator) {
+
+    var trelloApiCommand = {};
+
+    trelloApiCommand.makeTrelloApiCall = function(options, onComplete) {
+        if (!options.match) {
+            options.match = 'exact';
+            logger.warning('--match parameter not specified, defaulting to exact title match');
+        }
+        logger.info("Adding card");
+
+        // Grab our boards etc
+        var boardId = translator.getBoardIdByName(options.board);
+        var listId = translator.getListIdByBoardNameAndListName(options.board, options.list);
+
+        // Get a list of cards on a board
+        trello.get("/1/boards/" + boardId + "/cards/visible", function(err, data) {
+
+            if (err) {
+                throw err;
+            }
+            if (data.indexOf("Trello is down") !== -1) {
+                logger.error("Trello is currently unavailable");
+                process.exit(1);
+            }
+
+            var foundCards = [];
+            for (var i in data) {
+                var card = data[i];
+
+                // We only want cards from the list we specified
+                if (card.idList != listId) {
+                    continue;
+                }
+
+                if (options.match == 'exact') {
+                    if (card.name == options.title || !options.title){
+                            foundCards.push(card);
+                    }
+                } else if (options.match == 'contains') {
+                    if (card.name.includes(options.title)) {
+                            foundCards.push(card);
+                    }
+                }
+            }
+
```

Comment:
In other words, interactive prompting now becomes the default, with --force to just do it. This means the user would have to pass --force when he wants to pipe output somewhere, so as to avoid prompting?